app/apps/server/bridges/livechat.ts

```diff
@@ -87,15 +87,21 @@ export class AppLivechatBridge extends LivechatBridge {
 		return this.orch.getConverters()?.get('rooms').convertRoom(result.room);
 	}
 
-	protected async closeRoom(room: ILivechatRoom, comment: string, appId: string): Promise<boolean> {
+	protected async closeRoom(room: ILivechatRoom, comment: string, closer: IUser | undefined, appId: string): Promise<boolean> {
 		this.orch.debugLog(`The App ${ appId } is closing a livechat room.`);
 
-		return Livechat.closeRoom({
-			visitor: this.orch.getConverters()?.get('visitors').convertAppVisitor(room.visitor),
+		const closeData: any = {
 			room: this.orch.getConverters()?.get('rooms').convertAppRoom(room),
 			comment,
-			user: undefined,
-		});
+		};
+
+		if (closer?.id) {
+			closeData.user = this.orch.getConverters()?.get('users').convertById(closer.id);
```

Comment:
Where are you checking if the user with `close.id` exists? What if it doesn't?

Reply:
solved by the fix above